models/conflict.tsx

```diff
@@ -24,19 +25,21 @@ export default interface Conflict {
 }
 
 export enum ConflictResolutionType {
-  Trait1Wins = 0,
-  Trait2Wins,
+  Trait1None = 0,
```

Comment:
I think the order of `Trait1None` and `Trait2None` should be reversed here, otherwise it breaks backwards compatibility for conflict resolutions in existing collections.
eg. In the current version I have a conflict resolution set to `Trait1Wins`, meaning trait 2 gets dropped. If I use the same data in the new version, the conflict resolution gets changed to `Trait1None`, meaning trait 1 now gets dropped instead.